addons/website_profile/controllers/main.py

```diff
@@ -226,13 +225,41 @@ def view_all_users_page(self, page=1, **searches):
         users = User.sudo().search(dom, limit=self._users_per_page, offset=pager['offset'], order='karma DESC')
         user_values = [self._prepare_all_users_values(user) for user in users]
 
+        # Get karma position for users (only website_published)
+        position_domain = [('website_published', '=', True)]
```

Comment:
shouldn't it be the same domain than the initial one for the pager : `[('karma', '>', 1), ('website_published', '=', True)]`?

Reply:
Well, if we want to allow a user to find himself if he has 0 karma point, than, no, if not, because we consider he does not participate to the forum or elearning, then yes !

Reply:
No karma, no position, no blabla.